sql/7_0_2-to-7_0_3_upgrade.sql

```diff
@@ -112,4 +112,8 @@
 #IfMissingColumn form_encounter last_update
 ALTER TABLE `form_encounter` ADD `last_update` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;
 #EndIf
+#IfNotRow4D supported_external_dataloads load_type CQM_VALUESET load_source NIH_VSAC load_release_date 2023-05-04 load_filename ec_only_cms_20230504.xml.zip
+INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_release_date`, `load_filename`, `load_checksum`) VALUES
```

Comment:
This shouldn't be here, not sure how it even made it into this PR.

Reply:
This is one of those GitHub things. Not is my PR.